app/grandchallenge/evaluation/tasks.py

```diff
@@ -589,13 +652,17 @@ def update_phase_statistics():
     )
     phase_dict = {}
     for phase in phases:
-        avg_duration = get_average_job_duration_for_phase(phase)
-        average_algorithm_job_run_time = avg_duration.get(
+        duration_dict = get_average_job_duration_for_phase(phase)
+        average_algorithm_job_run_time = duration_dict.get(
             "average_duration", None
         )
-        accumulated_algorithm_job_run_time = avg_duration.get(
+        accumulated_algorithm_job_run_time = duration_dict.get(
             "total_duration", None
         )
+        monthly_costs = duration_dict.get("monthly_costs", None)
+        algorithms_submitted_per_month = duration_dict.get(
+            "algorithms_submitted_per_month", None
+        )
```

Comment:
When would the duration dict ever not contain those fields? The `None` is anyway unnecessary, I think the `get` is too.